src/components/LineChart/LineChart.jsx

```diff
@@ -311,75 +312,87 @@ class LineChart extends PureComponent {
         const goodYmax = yScale(incidentData[selectedASN][incIndex].goodPeriodMetric);
         const badYmax = yScale(incidentData[selectedASN][incIndex].badPeriodMetric);
 
+        const goodStart = incidentData[selectedASN][incIndex].goodPeriodStart;
+        const goodEnd = incidentData[selectedASN][incIndex].goodPeriodEnd;
+        const badStart = incidentData[selectedASN][incIndex].badPeriodStart;
+        const badEnd = incidentData[selectedASN][incIndex].badPeriodEnd;
+
         const goodWidth = xScale(incidentData[selectedASN][incIndex].goodPeriodEnd)
           - xScale(incidentData[selectedASN][incIndex].goodPeriodStart);
         const badWidth = xScale(incidentData[selectedASN][incIndex].badPeriodEnd)
           - xScale(incidentData[selectedASN][incIndex].badPeriodStart);
         const goodHeight = plotAreaHeight - goodYmax;
-        const badHeight = plotAreaHeight - badYmax;
-        const incidentHeight = Math.abs(badYmax - goodYmax);
-        // NOTE: This also must be manually tuned. It hides hover text in the case that the area is too small for the text to fit.
-        const rectFitsText = (goodWidth > 180) && (badWidth > 180);
+        // const badHeight = plotAreaHeight - badYmax;
+        // const incidentHeight = Math.abs(badYmax - goodYmax);
+        // Values chosen for aesthetic purposes
```

Comment:
Maybe make this comment for specific to what the values represent

Reply:
Added comments for variables